soc/riscv/riscv-privilege/telink_b91/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 # Copyright (c) 2021 Telink Semiconductor
 # SPDX-License-Identifier: Apache-2.0
 
+if (NOT DEFINED CONFIG_TELINK_B91_USB_SWI_ENABLE)
+	message(WARNING "USB SWI is enabled by CONFIG_TELINK_B91_USB_SWI_ENABLE")
```

Comment:
Condition and message seem so suspicious...

Reply:
and now ?

Reply:
I think it's better to warn user when flash is unprotected. like
if (CONFIG_TELINK_B91_USB_SWI_ENABLE)

Reply:
Done